shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -235,6 +241,7 @@ object GlutenConfig {
   val SPARK_HIVE_EXEC_ORC_COMPRESS: String = SPARK_PREFIX + HIVE_EXEC_ORC_COMPRESS
   val SPARK_SQL_PARQUET_COMPRESSION_CODEC: String = "spark.sql.parquet.compression.codec"
   val PARQUET_BLOCK_SIZE: String = "parquet.block.size"
+  val PARQUET_BLOCK_ROWS: String = "parquet.block.rows"
```

Comment:
Could you clarify where `parquet.block.rows` from? I could not find it in parquet-mr project.